xchange-blockchain/src/test/resources/__files/withdrawHistory-success.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "withdrawalId": "string",
+    "amount": 12.23,
+    "fee": 0.0005,
+    "currency": "BTC",
+    "beneficiary": "string",
+    "state": "REJECTED",
+    "timestamp": 1592830770594
+  }
+]
```

Comment:
to be consistent you should put these json files in a path where the deleted `address.json` was.

Reply:
Hi @timmolter first of all thanks for the review.
Regarding your comment, we put the json files in __files directory as recommended by WireMock documentation (“To read the body content of a file, place the file in the __files directory.” https://wiremock.org/docs/stubbing/).
We also saw that other integrations use the same approach. But anyway if you think we should change it, we can change it.